src/components/pages/rules/rules.js

```diff
@@ -43,6 +43,8 @@ export class Rules extends Component {
     }
 
     this.props.updateCurrentWindow('Rules');
+    this.props.logEvent(toDiagnosticsModel('RulesPage_Click', {}));
```

Comment:
Same comment as above.
You can remove this event if you want since I implemented this from navigation menu for all tabs.